blocks/author-bio-block/features/author-bio/default.jsx

```diff
@@ -63,207 +44,88 @@ const AuthorBioItemsContainer = () => {
-											<MediaLink webService="pinterest">
-												<PinterestIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-									case "snapchat":
-										socialButton = (
-											<MediaLink webService="snapchat">
-												<SnapchatIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-									case "whatsapp":
-										socialButton = (
-											<MediaLink webService="whatsapp">
-												<WhatsAppIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-									case "soundcloud":
-										socialButton = (
-											<MediaLink webService="soundcloud">
-												<SoundCloudIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-									case "rss":
-										socialButton = (
-											<MediaLink webService="rss">
-												<RssIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-									default:
-										socialButton = (
-											<MediaLink webService="email">
-												<EnvelopeIcon fill={PrimaryColor} />
-											</MediaLink>
-										);
-										break;
-								}
-								result.push(<Fragment key={socialLink.site}>{socialButton}</Fragment>);
-							}
-							return result;
-					  }, [])
-					: null;
-
-			// Make the name a hyperlink if a url to the bio page is provided
-			const authorName = original.byline ? (
-				<PrimaryFont as="h2" className="authorName" fontColor="primary-color">
-					{original.byline}
-				</PrimaryFont>
-			) : undefined;
-			const authorNameWithHyperlink = author.url ? (
-				<a href={author.url}>{authorName}</a>
-			) : undefined;
-
-			authorList.push(
-				<section key={author.name ? author.name : ""} className="authors">
-					<section className="author">
-						{renderAuthorInfo(author, arcSite)}
-						<section className="descriptions">
-							{authorNameWithHyperlink || authorName}
-							{/* there will always be a description via conditional on 52 */}
-							<p>{author.description}</p>
-							<section className="socialButtons">{socialLinks}</section>
-						</section>
-					</section>
-				</section>
-			);
-		}
-
-		return authorList;
-	}, []);
-
-	if (authors.length === 0) {
-		return null;
-	}
-
-	return (
-		<PrimaryFont as="section" className="author-bio">
-			{authors}
-		</PrimaryFont>
-	);
+		.filter(
+			({ additional_properties: additionalProperties = {}, description = "" }) =>
+				description?.length && additionalProperties?.original?.bio?.length
+		)
+		// A loop to generate the list of social media links.
+		// If no url is provided, then the social link will be skipped.
+		.map(
+			({
+				additional_properties: { original: { byline } } = {},
+				description,
+				name,
+				image: { url: imageUrl = "", alt_text: altText = "" } = {},
+				social_links: socialLinks = [],
+				url: authorUrl,
+			}) => (
+				<div className={`${BLOCK_CLASS_NAME}__author`} key={name}>
```

Comment:
```suggestion
				<Stack className={`${BLOCK_CLASS_NAME}__author`} key={name}>
```

Reply:
Thanks Jack. Fixed it in [96f9a90](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/96f9a90ab761f67fb76f87c84251162cdb2cc65e)

Reply:
It looks like I see display flex in the styles https://github.com/WPMedia/arc-themes-blocks/pull/1474/files#diff-bf5bbec7582ea0a26c4bd005ce1497053ed230cd3a1f1c4e4d0b43dba5b6dcbeR820

Flex wrap can also be updated to be done via the stack component. when we were going over this, I remember we talked about wrap and we talked about how this was in the components library

Reply:
sorry for also adding this clarification: flex-direction can also be replaced with a stack prop

Reply:
removed flex

Reply:
cool. thanks!